app/app.rb

```diff
@@ -0,0 +1,44 @@
+require 'sinatra/base'
+require './app/models/user.rb'
+require_relative 'data_mapper_setup'
+
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  set :session_secret, 'super secret'
+
+  get '/' do
+    erb :welcome_page
+  end
+
+  get '/registration' do
+    erb :registration
+  end
+
+  post '/registration' do
+    user = User.create(:username => params[:username], :email => params[:email], :password => params[:password])
```

Comment:
Metrics/LineLength: Line is too long. [112/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)